src/model/CarManager.js

```diff
@@ -0,0 +1,59 @@
+// @ts-check
+import Car from './Car.js';
+import pickRandomNumber from '../utils/pickRandomNumber.js';
+import OutputView from '../views/OutputView.js';
+import { CONFIG } from '../constants/config.js';
+
+class CarManager {
+  constructor() {
+    this.cars = [];
+  }
+
+  createCars(carNames) {
+    this.cars = carNames.map((carName) => new Car(carName));
+    return this.cars;
+  }
+
+  isMoveCondition(pickedRandomNumber) {
+    return pickedRandomNumber >= CONFIG.MINIMUM_RANDOM_NUMBER
+    && pickedRandomNumber <= CONFIG.MAXIMUM_RANDOM_NUMBER;
+  }
+
+  moveForwardCar(car, pickedRandomNumber) {
+    const moveCondition = this.isMoveCondition(pickedRandomNumber);
+    if (moveCondition) {
+      return car.move();
+    }
+  }
+
+  race(attempts) {
+    OutputView.printResultGreeting();
+    for (let i = CONFIG.ZERO; i < attempts; i++) {
+      this.cars.forEach((car) => {
+        this.moveForwardCar(car, pickRandomNumber());
+        this.showRaceResult(car.name, car.position);
```

Comment:
```suggestion
        OutputView.printRaceResult(car.name, car.position);
```
이렇게 바로 호출해도 괜찮을 것 같아요!